Tests/Responses/Transfer/ListTransferResponse.json

```diff
@@ -0,0 +1,55 @@
+{
+    "count": 23,
```

Comment:
data in the response is 2 so count should be 2 too

Reply:
Fixed